### PR TITLE
docs(platform-browser): add documentation for HammerGestureConfig

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #21146

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

Thank you, @vicb 👍 !!

### CODE REVIEW

File: packages/platform-browser/src/dom/events/hammer_gestures.ts [hunk 1]
```diff
@@ -64,14 +64,45 @@ export interface HammerInstance {
...
+* Use the `HammerGestureConfig` class to configure
+* [HammerJS](https://hammerjs.github.io/getting-started/),
+* an external library you can use with Angular to support gesture
+* recognition. The
+* [dependency injection token](guide/dependency-injection#dependency-injection-tokens)
+* should be bound to an instance of {@link   HammerGestureConfig}.
+* There are three key parts to the `HammerGestureConfig` class.
+*
+* @experimental
+*/
```
Comment (gkalpak) [inline packages/platform-browser/src/dom/events/hammer_gestures.ts]: Incorrect indentation 😱

File: packages/platform-browser/src/dom/events/hammer_gestures.ts [hunk 2]
```diff
@@ -64,14 +64,45 @@ export interface HammerInstance {
...
 @Injectable()
 export class HammerGestureConfig {
+
+  /**
+   * The `events` array is an array of strings that specifies the HammerJS
+   * [events](https://github.com/angular/angular/blob/master/packages/platform-browser/src/dom/events/hammer_gestures.ts#L15)
+   * you're supporting.
+  */
```
Comment (gkalpak) [inline packages/platform-browser/src/dom/events/hammer_gestures.ts]: Incorrect indentation 😱

File: packages/platform-browser/src/dom/events/hammer_gestures.ts [hunk 3]
```diff
@@ -64,14 +64,45 @@ export interface HammerInstance {
...
+
   events: string[] = [];
 
+  /**
+ * The `overrides` object is a map of recognizer name to configuration that the
+ * default implementation uses. For example, to disable the rotate gesture,
+ * you would use `{"rotate": {"enabled": false}}`.
+ */
```
Comment (gkalpak) [inline packages/platform-browser/src/dom/events/hammer_gestures.ts]: Incorrect indentation 😱

File: packages/platform-browser/src/dom/events/hammer_gestures.ts (hunk 1 above)
Comment (kapunahelewong) [reply packages/platform-browser/src/dom/events/hammer_gestures.ts]: Thank you. Is it better now? I've run `gulp format`.

Comment (gkalpak) [reply packages/platform-browser/src/dom/events/hammer_gestures.ts]: Unfortunately, `gulp format` does not affect the comments. Essentially, this is what it should look like

```
/**
 *   <-- GOOD
 *   <-- GOOD
 */  <-- GOOD

/**
 *   <-- BAD
 *   <-- BAD
 */  <-- BAD
```

**Comment by kapunahelewong** [comment]
@jbogarthyde is currently reviewing to make sure it's inline with her API template work.

**Comment by jenniferfell** [comment]
Moved to PR 22247 https://github.com/angular/angular/pull/22247

**Pull request closed without merging**